pypeit/core/extract.py

```diff
@@ -1170,7 +1170,7 @@ def parse_hand_dict(hand_extract_dict):
 
 
 def objfind(image, thismask, slit_left, slit_righ, inmask = None, FWHM = 3.0,
-            hand_extract_dict = None, std_trace = None, ncoeff = 5, nperslit = 10,  bg_smth = 5.0, pkwdth = 3.0,
+            hand_extract_dict = None, std_trace = None, ncoeff = 5, nperslit = None,  bg_smth = 5.0, pkwdth = 3.0,
```

Comment:
Why has this been changed from 10 to None. I have a hard time believing you have 10 objects on your slit? Is that really the case? Actually the thing to raise would be the sig_thresh